content/redirect.js

```diff
@@ -170,19 +162,26 @@ function removeConflictingOldRedirects(oldPairs, updatePairs) {
   return oldPairs.filter(([from, to]) => {
     const conflictingTo = newTargets.has(from.toLowerCase());
     if (conflictingTo) {
-      console.log(`removing conflicting redirect ${from}\t${to}`);
+      console.warn(
+        `Breaking 301: removing conflicting redirect ${from}\t${to}`
+      );
```

Comment:
I'm kinda sick and tired of this issue but perhaps an option would be to trigger an extra dialog/prompt if you're about to make this redirect. Like:

```
This will make what was previous a permanent redirect (301) a final destination. 
This might cause conflicts with old browser HTTP caches. Are you sure?
```

Reply:
I agree but would move it to another issue. I'm writing this up right now.